PowerShell/BuildingBlocks/Authentication/GetTicketForCustomerOrWorkgroup.ps1

```diff
@@ -0,0 +1,17 @@
+﻿# This script shows how to get an authentication ticket
+# 1) Scoped to a customer
+# 2) Scoped to a workgroup
+
+# Authenticate
+$creds = Get-Credential
+$ticket = Get-BT_Ticket -Credentials $credentials -ServiceType BitTitan
+
+# 1) Get a ticket for a customer
```

Comment:
-> Get a ticket scoped to a customer ?

"For a customer" makes it sound like this ticket can only be used for performing operations on a customer, which can be confusing to some people

Same below too for a workgroup I think

Reply:
It also stays consistent with what you wrote at the top level comments for the script

Reply:
Makes sense, fixed